### PR TITLE
Feat/eos core explain

### PR DESCRIPTION
**Pull request by gibrangul95**
@fdmirza explain transaction implementation and tests

### CODE REVIEW

File: modules/account-lib/src/coin/eos/transaction.ts [hunk 1]
```diff
@@ -36,10 +36,39 @@ export class Transaction extends BaseTransaction {
...
+      this._signatures.push(signature);
       this._signedTransaction?.signatures.push(signature);
     });
   }
 
+  async loadInputsAndOutputs(): Promise<void> {
+    const txJson = await this.toJson();
+    const actions = txJson.actions;
+    actions.forEach((action) => {
+      if (action.name === 'transfer') {
```
Comment (fdmirza) [inline modules/account-lib/src/coin/eos/transaction.ts:48]: What will be inputs/outputs for other actions?

File: modules/account-lib/src/coin/eos/transaction.ts [hunk 2]
```diff
@@ -36,10 +36,39 @@ export class Transaction extends BaseTransaction {
     const txHex = Buffer.from(this._signedTransaction.serializedTransaction);
     keys.forEach((key) => {
       const signature = ecc.Signature.sign(txHex, key.getKeys().prv).toString();
+      this._signatures.push(signature);
       this._signedTransaction?.signatures.push(signature);
     });
   }
 
+  async loadInputsAndOutputs(): Promise<void> {
```
Comment (fdmirza) [inline modules/account-lib/src/coin/eos/transaction.ts:44]: are we covering this in tests?

Comment (fdmirza) [reply modules/account-lib/src/coin/eos/transaction.ts:44]: Where is this method `loadInputsAndOutputs` used ?

**Review by fdmirza — CHANGES_REQUESTED** [review]

File: modules/account-lib/src/coin/eos/transaction.ts (hunk 2 above)
Comment (gibrangul95) [reply modules/account-lib/src/coin/eos/transaction.ts:44]: its not actually being used anywhere, the format of the transaction._outputs is different from that of explain transaction. Based on the current code we could remove this function.

**Review by fdmirza — APPROVED** [review]

**Pull request merged**